src/alchemlyb/parsing/amber.py

```diff
@@ -189,6 +203,32 @@ def file_validation(outfile):
             logging.warning('  WARNING: no free energy section found, ignoring file')
             invalid = True
 
+        mbar_ndata = 0
+        have_mbar, mbar_ndata = secp.extract_section('^FEP MBAR options:',
+                                                      '^$',
+                                                      ['ifmbar',
+                                                        'bar_intervall'],
+                                                      '^---')
+        if have_mbar:
+            mbar_ndata = int(nstlim / mbar_ndata)
+            mbar_lambdas = _process_mbar_lambdas(secp)
+            file_datum.mbar_lambdas = mbar_lambdas
+            clambda_str = '%6.4f' % clambda
+
+            if clambda_str not in mbar_lambdas:
```

Comment:
This block is not tested. See https://codecov.io/gh/alchemistry/alchemlyb/pull/51/src/src/alchemlyb/parsing/amber.py?before=src/alchemlyb/parsing/amber.py#L218

Put a `#pragma: no cover` to indicate that we exclude it from test.

Reply:
Should be addressed in [alchemistry/alchemtest#23](https://github.com/alchemistry/alchemtest/pull/23)